### DISCUSSION TITLE
Companies / Sites using Radix Primitives

### DISCUSSION THREAD

**Discussion by peduarte** [radix-ui/primitives#850] [Show and tell]
Are you using Radix Primitives for your site, or at your company? Share here!

To kick us off:
- CodeSandbox — https://codesandbox.io
- Vercel – https://vercel.com
- Livepeer - https://livepeer.org
- Basedash – https://www.basedash.com
- basement – http://basement.studio
- OpenFin – https://www.openfin.co
- Supabase – https://supabase.io

🚀 

Using Stitches too? [Share here](https://github.com/modulz/stitches/discussions/768)!

**Comment by itsdillon** [comment]
Radix is being used for some components in https://1upblitz.com

**Comment by braposo** [comment]
Using radix in the Managed Service product at https://www.singlestore.com/

**Comment by tarngerine** [comment]
just launched today: https://sprout.place/

read more: https://www.producthunt.com/posts/sprout-1-0

**Reply by colmtuite** [reply]
Gorgeous

**Comment by bukinoshita** [comment]
We've been using at Workos as well in our Dashboard. http://workos.com ❤️

**Comment by billsaysthis** [comment]
Several RadixUI components in production at https://www.act-on.com (in our app/service, not the marketing website).

**Comment by williamhzo** [comment]
using Radix primitives here @ [getmansa.com](https://getmansa.com/) and loving it ❤️ 
amazing work from the Modulz team 🙏

**Comment by frgarciames** [comment]
We built our UI library with stitches and radix
https://github.com/finetwork-os/ui
https://finetwork-ui.vercel.app/
And used in the private area of our users
https://mifi.finetwork.com/

**Comment by franklinjavier** [comment]
We're using here at @grupoboticario / @belezanaweb
Stitches + Radix-ui = Flora.ds

**Comment by stuarthendren** [comment]
https://committed.io/components
https://github.com/commitd/components

We are using radix-ui primitives and stitches for our design system that's used in many projects.

**Comment by tomerlichtash** [comment]
Just launched [Mel's Loop - A Comprehensive Guide to The Story of Mel](https://melsloop.com) which uses Radix's `Popover`, `Scroll`, `Menu`, `ToggleGroup` and `Icons` (and soon `Tooltip`). Loving it! Source code [here](http://github.com/tomerlichtash/mels-loop).

PS. Also using some stitches, although the entire project's approach is to avoid CSS in JS, so slowly migrating out of it in favor of a different solution.

**Comment by dcastil** [comment]
We at https://risecalendar.com are using Radix for our design system. 😊

**Comment by linkb15** [comment]
https://jungle.co we are using this in our website. good work guys. 😇

**Comment by tresabhi** [comment]
A static Astro app with quite the punch: https://blitzkit.app/

**Comment by maxmdev** [comment]
We use Radix on https://startintegrate.com. Really enjoy its plug&play.

**Comment by nmelo** [comment]
[Beadbox](https://beadbox.app) — a native desktop GUI for the [beads](https://github.com/steveyegge/beads) issue tracker. Built with Radix UI primitives via shadcn/ui (Dialog, Sheet, Table, Badge, DropdownMenu, Select, Tabs, Toast). The app runs Next.js 16 + React 19 inside a Tauri v2 native shell, shipping as a ~160MB binary on macOS, Linux, and Windows. We're building it with 8 AI coding agents working simultaneously, coordinated through beads itself.
